src/Uno.UI.Toolkit/RuntimeHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UI.Helpers;
+
+namespace Uno.UI.Toolkit;
+
+public static class PlatformRuntimeHelper
```

Comment:
It makes sense to have this helper to include it for winappsdk, but in that case, the usefulness of the `Uno.UI.Helpers` helper is very limited to uno users and could cause namespace and `#if` fiddling.

I see two ways here, either we :
1. Don't include that API for winappsdk, and let app developers `#if WINDOWS` or similar around it
2. Hide the `Uno.UI.Helpers` version from the public API, so it can still be used internally, and only keep the `Uno.UI.Toolkit` one.

The first one has the advantage of avoiding to have two versions seemingly of the same classes, but forces `#if` uses.

The second one has the advantage of being available everywhere, but is located in an unrelated namespace.

I think I prefer the second, just for the sake of developer convenience.